server/api/after_auth_handler.py

```diff
@@ -1,6 +1,7 @@
-from flask import jsonify, Blueprint, redirect, url_for, session, jsonify
+from flask import jsonify, Blueprint, redirect, url_for, session, jsonify, request, make_response
```

Comment:
jsonify is imported twice. Not sure if that will cause any issues or not

Reply:
fixed